lib/src/util/react_wrappers.dart

```diff
@@ -108,10 +109,34 @@ Expando<UnmodifiableMapView> _elementPropsCache = new Expando('_elementPropsCach
 /// For a JS component, this returns the result of [getJsProps] in an unmodifiable Map view.
 ///
 /// Throws if [instance] is not a valid [ReactElement] or composite [ReactComponent] .
-Map getProps(/* ReactElement|ReactComponent */ instance) {
+Map getProps(/* ReactElement|ReactComponent */ instance, {bool traverseWrappers: false}) {
```

Comment:
@jacehensley-wf the doc comment should be updated to explain the use case of setting `traverseWrappers` to `true`.